packages/salesforcedx-vscode-apex/package.nls.json

```diff
@@ -17,5 +17,8 @@
   "force_apex_test_last_method_run_text": "SFDX: Re-Run Last Run Apex Test Method",
   "force_apex_test_class_run_text": "SFDX: Run Apex Test Class",
   "force_apex_test_method_run_text": "SFDX: Run Apex Test Method",
-  "force_apex_test_run_text": "SFDX: Run Apex Tests"
+  "force_apex_test_run_text": "SFDX: Run Apex Tests",
+  "force_apex_test_suite_run_text": "SFDX: Run Apex Test Suite",
```

Comment:
please have @sbudhirajadoc review this

Reply:
the messages were approved beforehand in a separate doc! thanks sonal!